bin/filter_ambient_rna.r

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env Rscript
+
+library(SoupX)
+library(DropletUtils)
+
+# Load data and estimate soup profile
+soup_object <- load10X(getwd())
+
+# Estimate level of background contamination
+soup_object <- autoEstCont(soup_object)
```

Comment:
Like Scrublet, this may also fail with small samples. May want to switch to cellbender when it's released